templates/events/display.html

```diff
@@ -141,8 +181,8 @@ <h5 class="q-mt-none">Buy Ticket</h5>
               axios
                 .post(
                   '/events/api/v1/tickets/' +
-                  '{{ event_id }}/' +
-                  self.paymentCheck,
+                    '{{ event_id }}/' +
+                    self.paymentCheck,
```

Comment:
- why not use the template string here (``) as above (for consistency)?

Reply:
I'll refactor. I was just touching what was needed to escape the strings...